src/main/java/org/folio/circulation/resources/RequestNoticeSender.java

```diff
@@ -72,9 +72,7 @@ public RequestNoticeSender(PatronNoticeService patronNoticeService,
     patronNoticeService.acceptNoticeEvent(requestCreatedEvent, NoticeLogContext.from(request));
 
     Loan loan = request.getLoan();
-    if (request.getRequestType() == RequestType.RECALL &&
-      loan != null && loan.hasDueDateChanged()) {
-
+    if (request.getRequestType() == RequestType.RECALL && loan != null) {
```

Comment:
Should this logic have also changed for when a request has been moved (the method below)?